addons/jam/CfgWeapons.hpp

```diff
@@ -5,96 +5,96 @@ class CfgWeapons {
     class Launcher_Base_F;
 
     class mk20_base_F: Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
+        magazineWell[] += {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
     };
     class SDAR_base_F: Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG"};
+        magazineWell[] += {"CBA_556x45_STANAG"};
     };
     class Tavor_base_F: Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
+        magazineWell[] += {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D"};
     };
     class arifle_SPAR_01_base_F : Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D", "CBA_556x45_STANAG_2D_XL"};
+        magazineWell[] += {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D", "CBA_556x45_STANAG_2D_XL"};
     };
     class arifle_SPAR_02_base_F : Rifle_Base_F {
-        magazineWell[] = {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D", "CBA_556x45_STANAG_2D_XL"};
+        magazineWell[] += {"CBA_556x45_STANAG", "CBA_556x45_STANAG_L", "CBA_556x45_STANAG_XL", "CBA_556x45_STANAG_2D", "CBA_556x45_STANAG_2D_XL"};
     };
 
     class UGL_F : GrenadeLauncher {
-        magazineWell[] = {"CBA_40mm_M203", "CBA_40mm_EGLM"};
+        magazineWell[] += {"CBA_40mm_M203", "CBA_40mm_EGLM"};
     };
 
     class arifle_MX_Base_F : Rifle_Base_F {
-        magazineWell[] = {"CBA_65x39_MX", "CBA_65x39_MX_XL"};
+        magazineWell[] += {"CBA_65x39_MX", "CBA_65x39_MX_XL"};
         class GL_3GL_F : UGL_F {
-            magazineWell[] = {"CBA_40mm_3GL", "CBA_40mm_M203", "CBA_40mm_EGLM"};
+            magazineWell[] += {"CBA_40mm_3GL", "CBA_40mm_M203", "CBA_40mm_EGLM"};
```

Comment:
There is no 40mm magwell in vanilla.

Reply:
+= won't break if the array exists right? So it doesn't really matter

Reply:
Yep, but that's the thing, doesn't exist, same thing on the UGL_F parent.

Reply:
Last I checked this kind of array addition combined with inheritance was really finicky.
The above bits seem fine though, did a quick check and magazineWell is populated fine.